README.md

```diff
@@ -1,73 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+Project Name: Dario Alessio's Portfolio
 
-> Description the project.
+Description the project: 
+This is a page for mobile devices create to include professional information regarding technical and soft skills and abilities to inform potential employers.
 
+This is the first part of the whole project and so it includes only the homepage for mobile devices.
 
-## Built With
+Project will be updated in next weeks with more information about the completed projects that will form part of the portafolio.
```

Comment:
- [ ] [OPTIONAL] As you can see here the titles have a # before them, kindly add that so it will look more professional.